selfdrive/car/gm/carcontroller.py

```diff
@@ -28,11 +28,11 @@ def __init__(self, car_fingerprint):
     self.ADAS_KEEPALIVE_STEP = 10
     # pedal lookups, only for Volt
     MAX_GAS = 3072              # Only a safety limit
-    ZERO_GAS = 2048
+    self.ZERO_GAS = 2048
```

Comment:
Does it make sense to move it, MAX_GAS, and ZERO_GAS (making those self.xxx as well for consistency), to CarControllerParams' init, to CAR.VOLT block?

Reply:
Maybe, we really need another vehicle in here to determine if the GM actuator values are actually different from car to car.